sharding/nodesCoordinator/indexHashedNodesCoordinator.go

```diff
@@ -739,10 +751,15 @@ func (ihnc *indexHashedNodesCoordinator) computeNodesConfigFromList(
 			log.Debug("inactive validator", "pk", validatorInfo.PublicKey)
 		case string(common.JailedList):
 			log.Debug("jailed validator", "pk", validatorInfo.PublicKey)
+		case string(common.SelectedFromAuctionList):
+			if ihnc.flagStakingV4.IsSet() {
```

Comment:
add a log.Error - in case of "else"

Reply:
later after like refactoring - returning error here should be good as well.

Reply:
Decided to return error right here, actually.